packages/api/cms-api/src/dam/files/files.controller.ts

```diff
@@ -217,18 +220,28 @@ export function createFilesController({ Scope: PassedScope }: { Scope?: Type<Dam
                 });
             } else {
                 try {
-                    response = await this.blobStorageBackendService.getFile(this.damConfig.filesDirectory, createHashedPath(file.contentHash));
+                    stream = await this.blobStorageBackendService.getFile(this.damConfig.filesDirectory, createHashedPath(file.contentHash));
                 } catch (err) {
                     throw new Error(`File-Stream error: (storage.getFile) - ${(err as Error).message}`);
                 }
 
+                stream.on("error", (error) => {
+                    this.logger.error("Stream error:", error);
+                    res.end();
+                });
+
+                res.on("close", () => {
+                    this.logger.debug("Client disconnected");
```

Comment:
Will this be shown on default logger labels? If yes, should we configure the logger to only show this when we enable debug logs?

Reply:
Yes. To my understanding, Nest doesn't have a default configuration for log levels based on the NODE_ENV. Since we don't configure the log level anywhere, this is logged in all environments. 

I decided to remove the log. I don't think we really benefit from it. a53b9fdb100f106937a96bbdede8aebe08e804bd